src/main/java/fi/otavanopisto/kuntaapi/server/index/AbstractIndexHander.java

```diff
@@ -124,7 +126,9 @@ private InetSocketTransportAddress resolveTransportAddress(String name, int port
   
   private InetAddress resolveInetAddress(String name) {
     try {
-      return InetAddress.getByName(name);
+      InetAddress address = InetAddress.getByName(name);
+      System.out.println(String.format("IP: %s", address.getHostAddress()));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)